docs/pages/demo/datepicker/CustomDay.example.js

```diff
@@ -1,5 +1,5 @@
+import * as React from 'react';
 import clsx from 'clsx';
-import React, { useState } from 'react';
```

Comment:
I am still aware of changing extensions. Let's use `jsx` for now. 
Or please provide any arguments for changing it.

Reply:
Ok, will revert